packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade-request.html

```diff
@@ -0,0 +1,28 @@
+<form novalidate name="upgrade-form">
+    <oui-modal
+        data-heading="{{ 'dedicated_server_dashboard_upgrade_title_' + $ctrl.$stateParams.selectedUpgrade | translate }}"
+        data-primary-action="$ctrl.upgrade()"
+        data-primary-label="{{:: 'dedicated_server_dashboard_upgrade_submit_label' | translate }}"
+        data-primary-disabled="$ctrl.isLoading"
+        data-secondary-action="$ctrl.cancel()"
+        data-secondary-label="{{:: 'dedicated_server_dashboard_upgrade_cancel_label' | translate }}"
+        data-on-dismiss="$ctrl.cancel()"
+        data-loading="$ctrl.isLoading"
+    >
+        <p
+            data-translate="{{'dedicated_server_dashboard_upgrade_content_' + $ctrl.$stateParams.selectedUpgrade}}"
+            data-translate-values="{ 'projectId': $ctrl.$stateParams.productId }"
+        ></p>
+
+        <oui-field label="{{ $ctrl.issueTypeFieldsStr }}" size="xl">
```

Comment:
Why not directly translate the label and avoid creating a variable in the controller ? 
```suggestion
        <oui-field label="{{:: 'my_key' | translate  }}" size="xl">
```